Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -479,4 +479,170 @@ HRESULT _DWriteFontGetBoundingBoxesForGlyphs(
     });
 
     return ret;
-}
\ No newline at end of file
+}
+
+#pragma region CGFontCreateWithDataProvider helpers
+
+/**
+ * Custom implementation of IDWriteFontFileStream that implements Read in terms of an underlying CFDataRef
+ */
+class DWriteFontBinaryDataStream : public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, IDWriteFontFileStream> {
```

Comment:
InspectableClass is only available to WinRT types

---
In reply to: [88170542](https://github.com/Microsoft/WinObjC/pull/1386#discussion_r88170542) [](ancestors = 88170542)